pallets/credit/src/lib.rs

```diff
@@ -846,6 +848,24 @@ pub mod pallet {
                 }
             }
         }
+
+        fn init_delegator_history(account_id: &T::AccountId, era: u32) -> Weight {
+            let credit_data = Self::user_credit(account_id); // 1 db read
+            let weight = T::DbWeight::get().reads_writes(1, 0);
+            if credit_data.is_none() {
```

Comment:
Do we need to add one more condition that `credit_data.credit > 100`?

Reply:
no need，not judged in this